chain/sync.go

```diff
@@ -545,7 +545,13 @@ func (syncer *Syncer) ValidateBlock(ctx context.Context, b *types.FullBlock) err
 			return xerrors.Errorf("no candidates")
 		}
 
+		wins := make(map[uint64]bool)
```

Comment:
One small nit, is we can use `map[uint64]struct{}`.

Reply:
but results in slightly cleaner code to use the bool